.github/workflows/terraform-lint.yml

```diff
@@ -49,15 +49,10 @@ jobs:
           repository: 'abcxyz/pkg'
           path: 'abcxyz-pkg'
 
-      - id: 'setup-go'
-        uses: 'actions/setup-go@fac708d6674e30b6ba41289acaab6d4b75aa0753' # ratchet:actions/setup-go@v4
+      - id: 'lint'
+        uses: './.github/actions/terraform-linter'
```

Comment:
This needs to reference the action from `./abcxyz-pkg/` since its checked out in the step above.

Reply:
https://github.com/abcxyz/pkg/pull/169